android-components/components/support/rustlog/src/main/java/mozilla/components/support/rustlog/RustLog.kt

```diff
@@ -37,15 +39,15 @@ object RustLog {
      *
      * @param crashReporter [CrashReporting] instance used for reporting 'error' log messages.
```

Comment:
We could remove this line from KDoc since the param is not used anymore.

Reply:
Good catch.  I also removed the documentation about megazording, AFAIK there's no such thing as a "non-megazord" build anymore.